### DISCUSSION TITLE
discord icon

### DISCUSSION THREAD

**Discussion by mxdi9i7** [tailwindlabs/heroicons#361] [Ideas]

**Comment by koodeau** [comment]
I am also waiting for that one

**Comment by zefir-git** [comment]
After discord, what's next? 200 more social media (yes including the obscure ones, because why discriminate??). And then perhaps every big company. And then perhaps every random company.

And all of this is useful in Heroicons... how?

Just use your favourite search engine and surely you'll find that https://simpleicons.org/ is the project that maintains thousands of brand icons so that other iconpacks don;t need to convolute their selection.

**Reply by ryzamedia** [reply]
I disagree with you entirely. Anything over 150m active users is enough to warrant its inclusion...

**Reply by zefir-git** [reply]
> I disagree with you entirely. Anything over 150m active users is enough to warrant its inclusion...

Whose job will it be to monitor the number of active users on a given site? Simpleicons is specifically dedicated to brand icons, and the brand you requested is already maintained there.
